hack/checkha.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
```

Comment:
I was quite confused by the name `checksha.sh`, and thought it was a misspelling of `checksha`. Perhaps we can call it `check-hash.sh`?

Reply:
This may also warrant living in `hack/util/check-hash.sh` to reduce bloat of the `hack` dir.